test.q

```diff
@@ -4,7 +4,9 @@ pt:{-2 $[first[x]~`..err;err;fail][x;y]}
 i:{` sv"  ",/:` vs x}
 ge:{if[not P;n+:1;ns+:1;:(::)];v:.Q.trp[x;y;{(`..err;x,"\n",.Q.sbt 1#y)}];n+:1;if[not(1b~v)|(::)~v;pt[v](y;file)]}
 P:1;N:0;MM:0#`
+
```

Comment:
The changes outlined here are probably unnecessary. As an alternative this file could be downloaded from the embedpy repository (Where the original version of the file is located). This can be done within the `travis.yml` file before invoking `q test.q tests/ -q`
```
if [[ $TESTS == "True" && "x$OD" != "x" && "x$QLIC_KC" != "x" ]]; then
      q test.q tests/ -q;
```
by adding 
```
curl -fsSL -o test.q https://github.com/KxSystems/embedpy/raw/master/test.q
```
As seen in the automl directory [here](https://github.com/KxSystems/automl/blob/master/.travis.yml#L48)